8.0/README.md

```diff
@@ -0,0 +1,330 @@
+PHP 8.0 container image
+=======================
+
+This container image includes PHP 8.0 as a [S2I](https://github.com/openshift/source-to-image) base image for your PHP 8.0 applications.
+Users can choose between RHEL and CentOS based builder images.
```

Comment:
We can say CentOS Stream specifically, to not confuse readers.

Reply:
Addressed by https://github.com/sclorg/s2i-php-container/pull/347/commits/8f076621afa4c74b55f27a51f365b8824a934f6b